neurom/check/runner.py

```diff
@@ -126,13 +126,14 @@ def _check_file(self, f):
 
         try:
             data = load_data(f)
-            result &= self._check_loop(data, 'structural_checks')
-            nrn = fst_core.FstNeuron(data)
-            result &= self._check_loop(nrn, 'neuron_checks')
         except StandardError as e:
             L.error('Failed to load data... skipping tests for this file')
             L.error(e.message)
-            result = False
+            return False, {f: OrderedDict(('ALL', False))}
```

Comment:
I wrote my config.yaml wrong, but it was error out saying it was b/c the file couldn't load, and that was confusing.